### PR TITLE
SmartDashboard docs

### PR DESCRIPTION
**Pull request by AlyssaCote**
I added a smartdashboard.rst file to kickstart the SmartDashboard docs within SmartSim.

### CODE REVIEW

File: doc/smartdashboard.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+**********
```
Comment (al-rigazzi) [inline doc/smartdashboard.rst:2]: The decorator `**********` should be at least as long as the text which is enclosing

**Review by al-rigazzi — CHANGES_REQUESTED** [review]
Just a change requested, otherwise LGTM

File: doc/smartdashboard.rst (hunk 1 above)
Comment (ankona) [reply doc/smartdashboard.rst]: +1 - fixed!

**Review by al-rigazzi — APPROVED** [review]
LGTM!

**Review by ankona — COMMENTED** [review]
Moved main body of docs to other repo and tweaked SmartSim `make docks` to include those instead.

**Pull request merged**